src/main/java/com/latebox/server/report/ReportRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.latebox.server.report;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ReportRepository extends JpaRepository<Report, Long> {
+
+
+    List<Report> findByName(String name);
```

Comment:
Report entity does not have a name property. This line need to be removed.

Reply:
Thanks